app/controllers/users_controller.rb

```diff
@@ -1,7 +1,6 @@
 class UsersController < ApplicationController
-    def index
-      
-    end
+  def index
+
   end
```

Comment:
Пустые методы объявляются одной строкой
```suggestion
  def index; end
```